.github/workflows/pypi-publish.yml

```diff
@@ -1,7 +1,10 @@
 name: Publish distributions to TestPyPI and PyPI
 
-# Only activate if the release is published.
+# TestPyPI upload is scheduled in each weekday.
+# PyPI upload is only activated if the release is published.
 on:
+  schedule:
+    - cron: '0 15 * * *'
```

Comment:
Test push runs each day, not each weekday. I think it is OK but inconsistent with the PR title.

Reply:
I agree. I think we do not necessarily restrict the test release by dates.

Reply:
@HideakiImamura Do we need this commit (https://github.com/optuna/optuna/pull/2220/commits/dd1e71a47531b6fc8a45987040103e6aef2f7073)? 
I thought @not522 suggested that we can publish test-pypi release every day.

Reply:
I think either is fine. But, I'm not sure the notation `'0 15 * * ? *'` works as expected.

Reply:
Sorry for the confusion. I misunderstood the specification of `cron`. I revert the commit https://github.com/optuna/optuna/commit/dd1e71a47531b6fc8a45987040103e6aef2f7073 and change the PR title.